index.html

```diff
@@ -153,6 +154,71 @@ <h1 class="title">Skills</h1>
       </div>
     </div>
   </div>
+  <h1 class="title"> Projects </h1>
+  <div class="projects">
+    <div class="bodycards">
+      <!-- Cotejo Card -->
+      <div class="card">
+        <img class="image" src="./images/projects/CotejoApp.png" alt="CotejoApp">
+        <div class="info">
+          <h2 class="name">CotejoApp</h2>
+          <p class="texts">A funny application to schedule matchs of football.</p>
+        </div>
+        <div class="links">
+          <div class="urls">
+            <a href="https://github.com/Andres2D/cotejo-app-front" target="_blank">
+              <i class="fa-brands fa-github" style="color: black;"></i>
+            </a>          
+          </div>
+          <div class="urls">
+            <a Class="text-cards" href="https://cotejo-app-beta.netlify.app/login" target="_blank">
+              <i class="fa fa-external-link" aria-hidden="true" style="color: #006AFF;"></i>
+            </a>
+          </div>
+        </div>
+      </div>
+      <!-- Form Card -->
+      <div class="card">
+        <img class="image" src="./images/projects/Form.png" alt="Form">
+        <div class="info">
+          <h2 class="name">Form</h2>
+          <p class="texts">Form that adds data to a dynamic table.</p>
+        </div>
+        <div class="links">
+          <div class="urls">
```

Comment:
This seems to be redundant, you should use a single div with all the <a> tags inside.
Review how to improve it from the css.